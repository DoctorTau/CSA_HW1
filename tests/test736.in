585
-5043 -4990 -4947 9635 8653 9735 -1232 -7983 4215 3950 -7637 8295 2847 -6840 -2084 1939 -9489 5658 542 -9264 -8806 9326 -518 4802 1721 6081 -9868 -7214 2403 5567 -7727 2477 -2436 9845 -5666 -5184 2201 -5956 -8413 -4260 8800 -1770 9626 2 1347 -4955 8208 2215 -949 -3774 -6788 5689 -1301 -8071 7534 -9584 -4950 -9625 238 6790 8599 -1825 -8617 7795 3028 -6536 5787 5427 -40 -6825 2218 2497 -6878 -9180 9702 3747 3353 -8294 4207 7393 -6654 -5473 -723 -4105 -1670 3633 9924 4135 -6101 -3395 -4125 -5254 9992 7829 4594 -5708 -509 2473 -449 5496 -3859 -2388 -7586 -872 1342 1992 7368 1717 270 -7737 -6592 6366 4555 -8202 4360 -6986 3640 1970 6900 1622 2038 -2791 3509 -3982 4537 8249 150 7088 -4066 1323 -5022 8379 -8672 5726 5636 7730 1404 -5693 -5728 -1618 -8044 -9003 8430 -7266 -7959 -9568 -7287 -8659 441 -9173 7592 9357 -1349 1445 -8928 5624 2979 -4899 -7229 -6337 6915 6599 3913 122 -6009 2339 6761 7689 -3366 -9164 3933 -1399 8446 -6182 7151 -3268 785 -2401 -7269 8027 2615 -1226 -1994 3054 5488 2027 -8792 -9104 -4004 5772 1932 6557 -5949 972 -3353 -9095 698 5762 -2103 -9002 7013 -5005 8180 1841 6184 3873 -1268 -1248 -3023 -3768 -8080 1802 454 -3904 -8362 1632 537 2240 -8269 7484 -2569 -9375 -6885 7212 -2168 -3768 -5669 77 -7794 -8632 6304 -2099 -6941 5722 -836 300 -3969 -2930 -5179 7910 -8631 -6607 -1814 8300 8842 1763 4654 6236 4520 -9631 -1584 6707 4345 -4211 9014 6068 -4805 3985 8587 959 -1293 -6325 -755 -790 -8968 -9960 7909 -3816 8115 9787 -170 898 2772 7622 3690 4208 -8550 6812 -6678 -3186 -4898 -6463 -4867 -8503 -8469 9222 3066 -7322 3380 6160 2482 3559 -2760 6271 120 -9241 5911 -1188 3900 2130 1891 -9286 -272 6862 8408 -6530 -1904 4734 -4753 2892 2332 -1841 2442 -653 -7744 9670 -4642 5681 -5159 4621 -4788 -442 -3400 855 7491 -5668 -9379 -3773 491 -4228 -8386 -4899 3420 4057 9990 -4007 -837 7920 -8835 5816 532 -7634 5289 -449 7819 2430 2425 -179 -1819 3609 1621 -8728 4010 -9218 -3402 3299 5350 129 714 -8761 5484 4519 3920 921 4029 -1076 5441 -6954 -6422 -1648 3633 -3757 1151 5467 -4722 9262 -3459 -4136 8079 -526 -7062 7898 -8248 -1734 -5930 9390 217 9913 8570 -7298 999 6987 1207 -1015 7519 1113 5447 -9030 8499 -7817 -2777 -6342 -3251 -2516 -6304 -4795 -9619 4823 -2919 -6827 -5563 4232 -5661 -5285 2842 -60 2007 -7371 8953 8919 -2312 -5107 9540 7156 -261 -7606 6783 -1625 -8609 -1219 8469 3744 8124 -4536 5824 -7634 3420 6389 -5791 -1905 7762 -9045 -7604 -8348 -6941 2822 -8091 -4357 -826 3070 7718 -7237 -249 -2829 -3023 -2759 -9913 -9027 -287 -5227 -4121 -7590 1928 -6755 -4253 7300 -9960 -214 7656 6453 -430 7029 354 4966 1111 -6799 7578 -2788 7303 -9092 -4708 3732 -1565 102 -9693 -5601 1564 -5067 -3579 -5327 -4360 1793 7491 4149 -6845 -7053 5123 7442 -4819 -6006 -8379 7528 -5484 -9641 7504 9413 6505 2335 -5036 -3805 -1147 1576 -3457 7358 -3028 -5624 7081 7889 -261 9543 3630 -3328 3379 149 3429 2370 1043 8333 1465 1236 2015 -4938 134 4864 -4379 -8224 851 -8494 -2911 4911 -8013 7239 200 -2870 -2614 7255 3320 -6294 -8316 -8328 -8380 -3112 2141 9223 9751 -7520 7901 -4615 -1490 235 4633 7966 3385 5701 -2854 1944 7062 1387 7796 3610 -4948 -2573 -197 -2457 6019 -6644 -5228 4322 2081 9669 2345 -2845 -6113 2388 -789 