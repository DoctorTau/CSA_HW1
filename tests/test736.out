-4990 -1232 -6840 -2084 -9264 -8806 -518 -9868 -7214 -2436 -5666 -5184 -5956 -4260 -1770 -3774 -6788 -9584 -4950 -6536 -40 -6878 -9180 -8294 -6654 -1670 -5254 -5708 -2388 -7586 -872 -6592 -8202 -6986 -3982 -4066 -5022 -8672 -5728 -1618 -8044 -7266 -9568 -8928 -3366 -9164 -6182 -3268 -1226 -1994 -8792 -9104 -4004 -9002 -1268 -1248 -3768 -8080 -3904 -8362 -2168 -3768 -7794 -8632 -836 -2930 -1814 -1584 -790 -8968 -9960 -3816 -170 -8550 -6678 -3186 -4898 -7322 -2760 -1188 -9286 -272 -6530 -1904 -7744 -4642 -4788 -442 -3400 -5668 -4228 -8386 -7634 -8728 -9218 -3402 -1076 -6954 -6422 -1648 -4722 -4136 -526 -7062 -8248 -1734 -5930 -7298 -9030 -6342 -2516 -6304 -60 -2312 -7606 -4536 -7634 -7604 -8348 -826 -7590 -9960 -214 -430 -2788 -9092 -4708 -4360 -6006 -5484 -5036 -3028 -5624 -3328 -4938 -8224 -8494 -2870 -2614 -6294 -8316 -8328 -8380 -3112 -7520 -1490 -2854 -4948 -6644 -5228 